src/atom-polymer.ts

```diff
@@ -33,8 +33,9 @@ class AtomPolymer {
 
   deactivate() {
     this.subscriptions.dispose();
-    this.linter = null;
-    this.autocompleter = null;
+    this.editorService = null;
+    this.linter = new Linter();
+    this.autocompleter = new Autocompleter();
```

Comment:
This feels very counter-intuitive.  Need comment to explain why `deactivate` creates new things.

While this fixes the problem for the `setProjectPaths` call, I would expect the nullifying to be correct here and the initialization to new instances of Linter/Autocompleter to take place in the `activate` since activate is what triggered these *and* activate was in the stacktrace too.